storage/applicationinsights/src/main/java/zipkin/storage/applicationinsights/ApplicationInsightsSpanConsumer.java

```diff
@@ -68,6 +85,24 @@ public void accept(List<Span> spans) {
         }
         String res = gson.toJson(jsonElement);
         String msg = "{ \"Span\":" + res + "}";
+        //data model changes
+        telemetry.getContext().getOperation().setId(traceId);
```

Comment:
Yes, Shall I concatenate both strings while setting id? How are we going to retrieve back?